timesketch/frontend-ng/src/components/Explore/TimelineChip.vue

```diff
@@ -104,20 +104,42 @@ limitations under the License.
 
     <v-menu v-else offset-y :close-on-content-click="false" content-class="menu-with-gap" ref="timelineChipMenuRef">
       <template v-slot:activator="{ on }">
-        <v-chip v-on="on" :style="getTimelineStyle(timeline)" class="mr-2 mb-3">
-          <v-icon v-if="timelineStatus === 'fail'" left color="red"> mdi-alert-circle-outline </v-icon>
-          <span class="timeline-name-ellipsis">{{ timeline.name }}</span>
+        <v-chip  @click="toggleTimeline()" :style="getTimelineStyle(timeline)" class="mr-2 mb-3 pr-1 timeline-chip" >
+          <v-icon v-if="timelineFailed" left color="red"> mdi-alert-circle-outline </v-icon>
+          <v-icon v-if="!timelineFailed" left :color="timelineChipColor"> mdi-circle </v-icon>
```

Comment:
What do you think of making the timeline color spheres larger and match the border of the chip on the left side?

```suggestion
          <v-icon v-if="timelineFailed" left color="red" size="x-large"> mdi-alert-circle-outline </v-icon>
          <v-icon v-if="!timelineFailed" left :color="timelineChipColor" size="xx-large" class="ml-n3"> mdi-circle </v-icon>
```

Reply:
SG. Done.